### PR TITLE
Move .scss files from wwwroot folder to Styles folder of the Playground project (#1907)

### LINKED ISSUE #1907: Move `.scss` files from `wwwroot` folder of the Playground project
Currently, there are still some `.scss` files remaining in the wwwroot folder of the Playground project that should be moved to the root folder of `Styles`.

image

### PR DESCRIPTION
**Pull request by MohammadJavadTabari**
this closes #1907

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj [hunk 1]
```diff
@@ -12,6 +12,11 @@
         <Content Condition=" '$(BlazorMode)' == 'BlazorWebAssembly' " Remove="Pages\_Host.cshtml" />
         <Content Condition=" '$(BlazorMode)' == 'BlazorWebAssembly' " Remove="Pages\_Layout.cshtml" />
         <Content Remove="compilerconfig.json" />
+        <Content Include="Styles\abstracts\_fonts.scss" />
```
Comment (ysmoradi) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj:15]: You've to make sure that you understand every change that you've made and all changes are required in your opinion (":
Please revert changes in csproj file as these are not required at all
Note that I understand that sometimes visual studio make these changes

**Review by ysmoradi — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj (hunk 1 above)
Comment (MohammadJavadTabari) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj:15]: yeah, I see it a few minutes ago and I'm  fixing it

Comment (MohammadJavadTabari) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj]: it's done now

**Review by msynk — COMMENTED** [review]
why did you rename the `icon.fluent.scss` file to `_icon.fluent.scss`?
and also I think this file is redundant and should be removed.
@maryamhdr  & @Cyrus-Sushiant ?

**Comment by maryamhdr** [comment]
> why did you rename the `icon.fluent.scss` file to `_icon.fluent.scss`? and also I think this file is redundant and should be removed. @maryamhdr & @Cyrus-Sushiant ?

This file has been added to the playground project because of the `Icongraphy` page

**Comment by msynk** [comment]
@maryamhdr 
but it seems it's not used anywhere in the project

**Comment by msynk** [comment]
@MohammadJavadTabari 
any update?

**Comment by MohammadJavadTabari** [comment]
@msynk 
since "icons.fluent.scss" is not used in the playground project . I just remove it

**Comment by msynk** [comment]
@MohammadJavadTabari 
before removing the `icons.fluent.scss` file, plz wait for @maryamhdr response about this file usage.
and also plz resolve the conflict

**Comment by maryamhdr** [comment]
> @maryamhdr but it seems it's not used anywhere in the project

It seems removing this file causes the icons not to be rendered but let me check it again :)

**Comment by MohammadJavadTabari** [comment]
@msynk 
@maryamhdr 
ok, I'm waiting for you to check and tell me

**Comment by maryamhdr** [comment]
> icons.fluent.scss

the `icons.fluent.scss` file can be removed :')
@msynk

**Comment by msynk** [comment]
ok
@maryamhdr & @Cyrus-Sushiant 
plz review this PR

**Review by Cyrus-Sushiant — APPROVED** [review]

**Review by maryamhdr — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**